README.md

```diff
@@ -10,7 +10,7 @@
 
 <p align="center"><img src="/img/demo.gif?raw=true"/></p>
 
-Built to be jusT cOol 👌🦄 !
+Built to be just cool 👌🦄 !
```

Comment:
This is probably done intentionally.

Reply:
Okay, that should be a style then. I'm still waiting for the other corrections to be merged though.

Reply:
@madhavarshney is right